.circleci/config.yml

```diff
@@ -137,19 +137,29 @@ workflows:
           type: approval
           requires:
             - build
+
       - deploy:
-          name: staging_deploy
-          namespace: staging
+          name: staging_deploy_live1
+          namespace: staging-live1
```

Comment:
It seems weird to me to call this namespace now if we're not going to use it strictly for just namespaces. Maybe `environment` or something?

Reply:
Thanks, that's a really good suggestion.

Updated now to reflect this feedback